src/search/merge_and_shrink/labels.h

```diff
@@ -1,44 +1,63 @@
 #ifndef MERGE_AND_SHRINK_LABELS_H
 #define MERGE_AND_SHRINK_LABELS_H
 
+// TODO: rename this file to global_labels.h
```

Comment:
I'd do this now. preserving the diff for the review would be a reason to postpone this but in this case the classes in this file changed so much that the diff isn't useful anyway.